client/cmd/ps.go

```diff
@@ -118,14 +116,13 @@ func PsRestart(appID, target string) error {
 		return err
 	}
 
-	fmt.Printf("done in %ds\n", int(time.Since(startTime).Seconds()))
-
-	processes, _, err := ps.List(c, appID, c.ResponseLimit)
-	if err != nil {
-		return err
+	if len(processes) == 0 {
+		fmt.Println("Could not find any processes to restart")
```

Comment:
Hmm, well it's technically a failure so we could. Hadn't thought about it much. Do we usually exit non-zero elsewhere?